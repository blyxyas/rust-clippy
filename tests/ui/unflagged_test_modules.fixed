// run-rustfix
// compile-flags: --test
#![allow(unused)]
#![warn(clippy::unflagged_test_modules)]

fn main() {
    // test code goes here
}

#[cfg(test)]
mod tests {
    fn my_test() {}
}

#[cfg(test)]
mod test {
    fn my_other_test() {}
}
