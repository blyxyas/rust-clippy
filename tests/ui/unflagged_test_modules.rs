// run-rustfix
// compile-flags: --test
#![allow(unused)]
#![warn(clippy::unflagged_test_modules)]

fn main() {
    // test code goes here
}

#[cfg(test)]
mod tests {
    fn my_test() {}
}

mod test {
    fn my_other_test() {}
}
