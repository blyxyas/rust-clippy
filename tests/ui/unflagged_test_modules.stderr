error: this testing module isn't flagged with #[cfg(test)]
  --> $DIR/unflagged_test_modules.rs:15:1
   |
LL | mod test {
   | ^^^^^^^^
   |
   = note: `-D clippy::unflagged-test-modules` implied by `-D warnings`
help: add the attribute
   |
LL + #[cfg(test)]
LL ~ mod test {
   |

error: aborting due to previous error

